anchor-contributor/tests/anchor-contributor.ts

```diff
@@ -87,42 +79,112 @@ describe("anchor-contributor", () => {
       await dummyConductor.attestSaleToken(connection, orchestrator);
     });
 
-    it("Bridge Sale Token To Null Recipient", async () => {
-      // we need to simulate attesting the sale token on Solana.
-      // this allows us to "redeem" the sale token prior to sealing the sale
-      // (which in the case of this test means minting it on the contributor program's ATA)
-      //ait dummyConductor.attestSaleToken(connection, orchestrator);
+    it("Attest Accepted Token from Ethereum", async () => {
```

Comment:
Removed bridging part of test because it does not illustrate anything.

Added attesting token meta data so we can create wrapped and complete transfer using the token bridge program. This token bridge wrapped token is the first accepted token and is used throughout the test.